packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -200,9 +156,9 @@ export class DebugSession implements Disposable {
         request: "attach",
         breakpointsAreRemovedOnContextCleared: isUsingNewDebugger ? false : true, // new debugger properly keeps all breakpoints in between JS reloads
```

Comment:
NIT: While we're here, we can probably remove this (or just set to `true`), since we only use the old debug adapter with the old debugger, and the new debug adapter doesn't check this flag.